src/modules/advertising.json

```diff
@@ -0,0 +1,69 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "Advertising",
+        "description": "A module for managing Advertisement settings.",
+        "version": "0.0.0"
+    },
+    "methods": [
+        {
+            "name": "resetIdentity",
```

Comment:
Maybe call this resetIdentifier. IFA isn’t exactly an identity and it’s a bit spooky to imply it is.